website/docs/r/cognito_user_pool.markdown

```diff
@@ -221,7 +221,10 @@ In addition to all arguments above, the following attributes are exported:
 
```

Comment:
Also add documentation for `custom_email_sender`, `custom_sms_sender` and `kms_key_id` in `lambda_config`.

Reply:
added